src/app/formFields/getAddressCustomFieldsValidationSchema.spec.tsx

```diff
@@ -1,14 +1,13 @@
 import { createLanguageService, LanguageService } from '@bigcommerce/checkout-sdk';
 import { ObjectSchema, ValidationError } from 'yup';
 
+import { getFormFields } from '../address/formField.mock';
 import { getShippingAddress } from '../shipping/shipping-addresses.mock';
 
-import { getFormFields } from './formField.mock';
-import getAddressCustomFieldsValidationSchema from './getAddressCustomFieldsValidationSchema';
-import getAddressValidationSchema from './getAddressValidationSchema';
-import { AddressFormValues } from './mapAddressToFormValues';
+import getCustomFormFieldsValidationSchema from './getCustomFormFieldsValidationSchema';
+import getFormFieldsValidationSchema, { FormFieldValues } from './getFormFieldsValidationSchema';
 
-describe('getAddressCustomFieldsValidationSchema', () => {
+describe('getCustomFormFieldsValidationSchema', () => {
```

Comment:
I think the name of the spec file is still called `getAddressCustomFieldsValidationSchema`. Should probably rename that